src/MeMoMetabolite.py

```diff
@@ -35,12 +35,18 @@ def __init__(
             orig_ids: list[str]|None = None,
             _model_id: str|None = None,
             names: list[str]|None = None,
+            names_source: list[str]|None = None,
             _inchi_string: str|None = None,
+            _inchi_source: str|None = None,
             _formula: str|None = None,
             _charge: int|None = None,
+            # TODO _charge_source: str|None = None,
```

Comment:
Not yet, however, this super low priority, because we don't use that whatsoever. I implemented it in the first place because of the prospect to use it when determining the correct protonation state of a metabolite in the common compartment. 1. we don't use these for now, 2. only modelseed provides this information (thus could be a fixed value as of now).